src/main/catalog-sources/kubeconfig-sync.ts

```diff
@@ -248,17 +255,25 @@ async function watchFileChanges(filePath: string): Promise<[IComputedValue<Catal
     depth: stat.isDirectory() ? 0 : 1, // DIRs works with 0 but files need 1 (bug: https://github.com/paulmillr/chokidar/issues/1095)
     disableGlobbing: true,
   });
-  const rootSource = new ExtendedObservableMap<string, ObservableMap<string, RootSourceValue>>(observable.map);
+  const rootSource = new ObservableMap<string, ObservableMap<string, RootSourceValue>>();
```

Comment:
Please revert unless my class doesn't work for some reason. This change is extraneous.

Reply:
@Nokel81 what the purpose for that extendable map over normal? @jakolehm wdyt?

Reply:
Reverted